container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -168,6 +168,38 @@ instance_groups:
     release: scf
   - name: patch-properties
     release: scf-helper
+  - name: eirini-persi-broker
```

Comment:
This needs feature: `eirini`

Reply:
Should be covered by: https://github.com/SUSE/scf/blob/e2aca801407febacf5d0174fcc04aa672eb1ca47/container-host-files/etc/scf/config/role-manifest.yml#L157